contracts/LosslessGovernance.sol

```diff
@@ -545,6 +545,24 @@ contract LosslessGovernance is Initializable, AccessControlUpgradeable, Pausable
 
     }
 
+    /// @notice This lets an erroneously reported account to retrieve compensation
+    function retrieveCompensationForContract(address token) public whenNotPaused {
```

Comment:
This will work only for LERC20 contracts. Maybe we could do something that works for all contracts. A lot of contracts implement Ownable, maybe we could use that. Also this function is missing check if address is actually a contract. 

function isContract(address addr) returns (bool) {
  uint size;
  assembly { size := extcodesize(addr) }
  return size > 0;
}

Reply:
So in this case what could happen is that someone could report and some LP Mining / Staking contract or any other DeFi contract that interacts with LERC20. If this contract is not whitelisted it would basically stop functioning after it is reported. That could be expected behaviour in cases where it’s being hacked. But it could also be done maliciously. So we could think of a way to pay a compensation for invalid freezing of the protocol. However, I think there’s no simple solution of how this could be done. So I would suggest leaving this problem for the v4 and not increase the scope of v3.

Reply:
Dismissed for V3 on commit bb7fe10